.github/workflows/performance.yml

```diff
@@ -50,16 +47,23 @@ jobs:
         run: uv run tests/performance/run.py --output-file $RUNNER_TEMP/performance.json
       - name: Download historic performance data
         if: github.event_name == 'pull_request'
-        run: gh gist clone ${{ env.gist_id }} $RUNNER_TEMP/historic
+        uses: actions/checkout@v4
+        with:
+          repository: gchq/coreax-metadata
+          path: tmp_coreax-metadata
```

Comment:
Is this still intended to be in a "tmp" folder, or is that a development artifact?

Reply:
Comment added.

Reply:
Ah, I think I misunderstood what the `path` option was for (I was imagining it was some kind of sparse checkout). The comment is helpful!